src/reducers/playlistReducer.js

```diff
@@ -0,0 +1,27 @@
+export const playlistReducer = (state, action) => {
+  // const index = state.findIndex((el) => el._id === action.payload._id);
```

Comment:
suggestion - avoid pushing commented code

Reply:
yes made the changes.